sourced/ml/cmd_entries/repos2ids.py

```diff
@@ -1,24 +1,25 @@
 import logging
+from typing import NamedTuple
 from uuid import uuid4
 
-from sourced.ml.transformers import Ignition, ContentExtractor, ContentDeserializer, \
-    Content2Ids, HeadFiles, Cacher
+from sourced.ml.transformers import Ignition, \
+    Content2Ids, ContentExtractor, HeadFiles, Cacher
 from sourced.ml.utils import create_engine, EngineConstants
 
 
 def repos2ids_entry(args):
     log = logging.getLogger("repos2ids")
     engine = create_engine("repos2ids-%s" % uuid4(), **args.__dict__)
-    documents_column_name = [EngineConstants.Columns.RepositoryId,
-                             EngineConstants.Columns.PathId]
-    ids_transformer = Content2Ids(args, documents_column_name)
+    Column = NamedTuple("Column", [("repo_id", str), ("file_id", str)])
+    language_mapping = Content2Ids.build_mapping()
```

Comment:
This should be a static and lazily created variables in Content2Ids - that is, initialized at the first request.